src/var/mash.jl

```diff
@@ -0,0 +1,34 @@
+# MinHash
+# ========
+#
+# Functions to MASH distance for MinHash sketches
+#
+# see DOI: 10.1186/s13059-016-0997-x
+#
+# This file is a part of BioJulia.
+# License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+function jaccarddistance(sketch1::MinHashSketch, sketch2::MinHashSketch)
+    d = length(setdiff(sketch1.sketch, sketch2.sketch))
```

Comment:
@bicycle1885 I'm not sure about merge sort, but do you mean something like: Iterating through the arrays independently in a loop, and since you know they're sorted you know based greater than, less than and so on, whether elements are in array A and are not in array B?

@kescobo Note that setdiff gets the values that are in sketch1 and not sketch2, but not the other way around, values in sketch2 that are not in sketch1 will not be returned. If this is what you want all is well, but I thought I'd mention it just in case.